BAZProject/Box.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Box.swift
+//  BAZProject
+//
+//  Created by Manuel S on 18/04/23.
+//
+
+import Foundation
+
+//MARK: Box para manejo de MVVM
+public struct Box<T> {
```

Comment:
Tsss excelente uso del Observer Pattern. Quizás sólo sería bueno agregar in-line comments en las propiedades y métodos de la estructura. Recuerda usar `///`